did-utils/src/methods/did_peer/method.rs

```diff
@@ -28,7 +25,7 @@ lazy_static::lazy_static!(
 const MULTICODEC_JSON: [u8; 2] = [0x80, 0x04];
 
 #[derive(Default)]
-pub struct DIDPeerMethod {
+pub struct DidPeer {
     /// Key format to consider during DID expansion into a DID document
     pub key_format: PublicKeyFormat,
```

Comment:
It seems to be your intent to also remove the pub qualifier here.
```suggestion
    key_format: PublicKeyFormat,
```

Reply:
My bad, let me remove it. Thanks